ui_components/src/travel/Map.js

```diff
@@ -41,8 +41,8 @@ export const Map = ({ mapData, scoutData, playerId, ranksToView, strategicView,
        How do we calculate the starting coordinate in this example? We need to offset the first visible tile by +2 which
        is equal to player X - stage midpoint X.
      */
-    const stage_offset_x = player_x - stage_midpoint_x;
-    const stage_offset_y = player_y - stage_midpoint_y;
+    const stage_offset_x = player_x - stage_midpoint_x - 1;
+    const stage_offset_y = player_y - stage_midpoint_y - 1;
```

Comment:
Huh, why is this needed?

Reply:
The player icon is incorrectly centered on live, this shifts it so its directly in the middle